client/cypress/support/app.po.ts

```diff
@@ -15,7 +15,7 @@ export class AppPage {
     return cy.get('.sidenav');
   }
 
-  getNavLink(navOption: 'Home' | 'Users') {
-    return cy.contains('[routerlink] > .mat-list-item-content', `${navOption}`);
+  getNavLink(navOption: ' Home ' | ' Users ') {
```

Comment:
Why do you put spaces around these? That seems odd, and not something I would expect people (students) to do in the future? Is there a reason for it?

Reply:
I wonder if this happened with some of the automated tooling for moving to Angular 15? Or, possibly @jpwalbran made a choice here?